config/intellij-idea-inspections.properties

```diff
@@ -12,4 +12,9 @@ idea.exclude.patterns=.idea/**;\
               src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/ParseTreeBuilder.java;\
               src/test/java/com/puppycrawl/tools/checkstyle/grammar/javadoc/ParseTreeBuilder.java;\
               config/idea.properties;\
-              config/intellij-idea-inspections.properties
+              config/intellij-idea-inspections.properties;\
+              .ci-temp/**;\
+              .classpath;\
+              .project;\
+              .settings/**;\
+              bin/**
```

Comment:
these were added to make sure that scope in project matches this file, since we cannot specify scope from cli.